api/views/phones.py

```diff
@@ -485,6 +484,24 @@ def inbound_call(request):
     )
 
 
+@decorators.api_view(["POST"])
+@decorators.permission_classes([permissions.AllowAny])
+def voice_status(request):
+    called = request.data.get("Called", None)
+    call_status = request.data.get("CallStatus", None)
+    if called is None or call_status is None:
+        raise exceptions.ValidationError("Call data missing Called, CallStatus")
+    if call_status != "completed":
+        return response.Response(status=200)
+    call_duration = request.data.get("CallDuration", None)
+    if call_duration is None:
+        raise exceptions.ValidationError("completed call data missing CallDuration")
+    relay_number, _ = _get_phone_objects(called)
+    relay_number.remaining_seconds = relay_number.remaining_seconds - int(call_duration)
+    relay_number.save()
```

Comment:
small chance of a race condition here, if multiple `voice_status` requests come in at the same time. For example, Twillio could queue up some status calls due to some pause on their side, and send them all at once. However, it would be in the user's favor - less seconds would be deducted than planned. 👍

Also, any errors result in a server error, but what else should we do, other than send the error to Sentry? I tested, and Twillio does not seem to retry on a server error.